lib/config.js

```diff
@@ -80,10 +80,15 @@ const conf = convict({
     }
   },
   expiration: {
-    code: {
+    accessTokenS: {
```

Comment:
I don't know if we use this convention anywhere else. I think we try to use milliseconds whenever we can, but the spec claims that the `expires_at` value should be in seconds...